tests/h/views/activity_test.py

```diff
@@ -8,7 +8,7 @@
 
 from h.views import activity
 
-@pytest.mark.usefixtures('paginate', 'query')
+@pytest.mark.usefixtures('paginate', 'query', 'pyramid_request_mocked')
```

Comment:
We do something like this in a few other places in the codebase except the usual pattern is to "override" the `pyramid_request` fixture for a suite of tests using something like this:

```py
class TestSomething(object):
  def test_it_does_something(self, pyramid_request):
    ...
  @pytest.fixture
  def pyramid_request(pyramid_request):
    # Customize request for this suite of tests
    return pyramid_request
```